components/GoodsCard.vue

```diff
@@ -1,24 +1,23 @@
 <template>
-  <div class="goods-card inline-block">
-
-    <div class="goods-content">
-      <div class="goods-img-container">
-        <img class="goods-image" :src="content.jpeg">
-        <div class="middle">
-          <div class="pencil-container">
+  <div class="goods-card">
+    <div class="goods-card_content">
+      <div class="goods-card_img-container">
+        <img class="goods-card_image" src="~/assets/img/car.jpg"/>
+        <div class="goods-card_overlay">
+          <div class="goods-card_pencil-container">
             <a href="#">
-              <span class="ellipse1" v-html="content.svg"></span><span class="pencilIcon" v-html="pencilIcon.svg"></span>
+              <PencilEdit class="goods-card_pencil-icon"/>
             </a>
           </div>
-          <div class="rubbishBin-container">
+          <div class="goods-card_bin-container">
             <a href="#">
-              <span class="ellipse2" v-html="content.svg"></span><span class="rubbishBinIcon" v-html="rubbishBinIcon.svg"></span>
+              <RubbishBin class="goods-card_bin-icon"/>
             </a>
```

Comment:
Usually, I combine BEM and Tailwind.
I mean use BEM classes only when necessary and only when you have custom styles that difficult to handle with tailwind.